Sources/DistributedActors/Cluster/Downing/DowningStrategy.swift

```diff
@@ -33,88 +32,93 @@ public protocol DowningStrategy {
 public enum DowningStrategyDirective {
     case none
     case markAsDown(Set<Cluster.Member>)
-    case startTimer(key: TimerKey, message: DowningStrategyMessage, delay: TimeAmount)
+    case startTimer(key: TimerKey, member: Cluster.Member, delay: TimeAmount)
     case cancelTimer(key: TimerKey)
 
     static func markAsDown(_ member: Cluster.Member) -> Self {
         Self.markAsDown([member])
     }
 }
 
-public enum DowningStrategyMessage: NonTransportableActorMessage {
-    case timeout(Cluster.Member)
-}
-
 // ==== ----------------------------------------------------------------------------------------------------------------
 // MARK: Downing Shell
 
-internal struct DowningStrategyShell {
-    typealias Message = DowningStrategyMessage
-    var naming: _ActorNaming = "downingStrategy"
+internal distributed actor DowningStrategyShell: DistributedActor {
```

Comment:
No need to type out the conformance, it should be implicitly there -- did you see problems with it?

Reply:
nope, didn't know.

Reply:
Ok cool, was worried if some bug 😂